internal/host/validator.go

```diff
@@ -841,3 +841,85 @@ func (v *validator) printDefaultRoute(c *validationContext, status ValidationSta
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *validator) isDomainNameResolvedCorrectly(c *validationContext) ValidationStatus {
```

Comment:
Since this is specifically testing the API domain validation, the function should be called IsApiDomainNameResolvedCorrectly